dataframe-excel/src/main/kotlin/org/jetbrains/kotlinx/dataframe/io/xlsx.kt

```diff
@@ -195,6 +218,29 @@ public fun DataFrame.Companion.readExcel(
     return dataFrameOf(columns)
 }
 
+/**
+ * This is a universal function for name repairing
+ * and should be moved to the API module later,
+ * when the functionality will be enabled for all IO sources.
+ */
+private fun repairNameIfRequired(nameFromCell: String, columnNameCounters: MutableMap<String, Int>, nameRepairStrategy: NameRepairStrategy): String {
+    return when(nameRepairStrategy) {
+        NameRepairStrategy.NO -> nameFromCell
+        NameRepairStrategy.CHECK_UNIQUE -> if (columnNameCounters.contains(nameFromCell)) throw DuplicateColumnNamesException(columnNameCounters.keys.toList()) else nameFromCell
+        NameRepairStrategy.MAKE_UNIQUE -> if (nameFromCell.isEmpty()) { // probably it's never empty because of filling empty column names earlier
+            val emptyName = "Unknown column"
+            if(columnNameCounters.contains(emptyName)) "${emptyName}_${columnNameCounters[emptyName]}"
```

Comment:
linting

Reply:
Also not sure whether we should use a `_`. In Kotlin they are only used for ENUM_NAMES and constants, whereas column names will be used as accessors. I think we can drop the `_` and just put the number after it. wdyt?

Reply:
I thought about it, it became uglier and badly readable, but let it be.
From other side, column names are not a part of Kotlin, but became part when we generate schemas

Reply:
Yeah, but we mainly push in the column accessor / DataSchema API, due to its type safety etc., so shouldn't we mainly focus on that? Especially also with the [camelCase issue](https://github.com/Kotlin/dataframe/issues/302) we had lately, and with  [naming convention concerns](https://github.com/Kotlin/dataframe/issues/383). I think we should play it safe and not become looking like python ;)